docs/source_values.rst

```diff
@@ -0,0 +1,133 @@
+=============
+Source/Values
+=============
+
+.. currentmodule:: gpiozero
+
+GPIO Zero provides a method of using the declarative programming paradigm to
+connect devices together: feeding the values of one device into another, for
+example the values of a button into an LED:
+
+.. literalinclude:: examples/led_button.py
+
+which is equivalent to:
+
+.. literalinclude:: examples/led_button_loop.py
+
+Every device has a ``.value`` property (the device's current value). Input
+devices can only have their values read, but output devices can also have their
+value set to alter the state of the device::
+
+    >>> led = PWMLED(17)
+    >>> led.value  # LED is initially off
+    0.0
+    >>> led.on()  # LED is now on
+    >>> led.value
+    1.0
+    >>> led.value = 0  # LED is now off
+
+Every device also has a ``.values`` property (a generator continuously yielding
+the device's current value). All output devices have a ``.source`` property
+which can be set to any iterator. The device will iterate over the values
+provided, setting the device's value to each element at a rate specified in the
+``source_delay`` property.
+
+.. image:: images/source_values.*
+
+The most common use case for this is to set the source of an output device to
+the values of an input device, like the example above. It is also possible to
+set the source of an output device to the values of another output device (so
+they match):
```

Comment:
Looks like this second sentence should be removed, as it's repeated a bit below?